src/components/map/KoorMap.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+    <div
+        id="map"
+        ref="map"
+        tabindex="1"
+        class="olmap"
+    >
+    </div>
+    <Menu class="koor-menu" :cover-area="coverArea"/>
```

Comment:
Jeg ville nok lade `KoorMenu` og `KoorMap` stå side om side (i en ny over-komponent, så vue-router kan håndtere det) i stedet for at lave en hierarkisk afhængighed her. 
`coverArea` kan de kan synkronisere via Store.

Reply:
God Ide :) Er gjort